### ISSUE TITLE
Tag each structure with a city

### ISSUE THREAD

**Issue by michaellevy** [michaellevy/FireAtTheFringe#2]

**Comment by pjbitterman** [comment]
just want to be clear - you want a "city" attribute on each house I output? This should be doable... I looked at this awhile back, and there's a bit of an issue when it comes to determining city bounds.

**Comment by michaellevy** [comment]
Yeah, the statistical model needs a city for each house to predict its ds
behavior. Some are not going to be in the seven cities we defined, which is
fine, but we need a city for them too.

**Comment by pjbitterman** [comment]
ok, I looked back at my data (it had been awhile). I can join the points back to the original parcels from which they were defined. The parcel layer has a city attribute, which I can export. Unless you need it right away, I'd rather wait until we have a proper WUI definition, though, so I don't have to do it twice.

**Comment by michaellevy** [comment]
No problem.

**Comment by pjbitterman** [comment]
turns out I already had it - just need a function definition of the WUI. I added you as a collaborator to the repo I was using, but the data is rather large >100MB, so it's in the gitignore.

There will need to be some cleanup on the city names, I'll looking into packages that can help do so programmatically.
